tests/test_vacuum_importance_sampling.py

```diff
@@ -15,7 +15,7 @@ def get_ff_am1ccc():
     return ff
 
 
-@pytest.mark.nogpu
```

Comment:
It doesn't look like there is a `nightly_no_cuda` CI stage, does this run?

Reply:
Good catch. I suppose this could be changed back to a `nogpu` test, but for consistency I opted to introduce a `nocuda_nightly_tests` target in https://github.com/proteneer/timemachine/pull/1108/commits/5e798c90b3d5aab08a4ab099d2c02e1c144f2f47.